### ISSUE TITLE
Behaviour of comment blocks when zooming / dezooming

### ISSUE THREAD

**Issue by jcelerier** [ossia/score#492]
Currently they keep the same size, which is not very practical. What should happen ? Maybe putting them in a "hidden popup" mode when they are too small ?

**Comment by pchdev** [comment]
yeah that's a tough one, it has been bothering me for a long time too, because the position of the comment gets messed up every time
there's not a lot of examples of comments in traditional sequencers I think.. well, at least not directly on the timeline... they usually implement the markers / regions stuff, or they just move them to the inspector. 
I really like having a sort of "comment track" on top of the timeline, with thin, almost transparent vertical lines pointing to the desired position, like a caption more or less.

but for now, yes, maybe your solution is best, I don't have too many ideas on this.
